src/components/Comments/WriteComment.js

```diff
@@ -98,7 +99,7 @@ const WriteComment = ({
         {isAnonimity ? (
           <>
             {loadingUser ? (
-              <ActivityIndicator size={'small'} color={COLORS.holyTosca} />
+              <ActivityIndicator size={'small'} color={colors.bondi_blue} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `ActivityIndicator` has been changed. Make sure this color change aligns with your design guidelines.
```diff
- <ActivityIndicator size={'small'} color={COLORS.holyTosca} />
+ <ActivityIndicator size={'small'} color={colors.bondi_blue} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->